src/pages/index.js

```diff
@@ -38,6 +57,26 @@ export const query = graphql`
         }
       }
     }
+    goodbye: allMarkdownRemark(filter: { fileAbsolutePath: { regex: "/homepage/goodbye.md$/" } }) {
```

Comment:
what's goodbye.md?

Reply:
ahh this was a previous file that I don't use anymore, will remove it.